vault/mount.go

```diff
@@ -92,6 +94,7 @@ var (
 		systemMountType,
 		"token",
 		identityMountType,
+		pluginMountType,
```

Comment:
I thought we got rid of the "plugin" type, why is this a singleton mount now?

Reply:
Yeah i think this should be removed from this list

Reply:
Well, the main reason it's not fully gone is because the "plugin" backend's Factory and New method are still used to mount the other backends as plugins, so we still need the one. However, we don't want people to mount additional versions of it.